test/contrib/test_autoguide.py

```diff
@@ -32,13 +34,16 @@ def body_fn(i, val):
         loss, opt_state_, rng_ = svi_update(i, rng_, opt_state_, model_args=(data,), guide_args=(data,))
         return opt_state_, rng_
 
-    opt_state, _ = lax.fori_loop(0, 300, body_fn, (opt_state, rng_train))
-    median = guide.median(opt_state)
-    assert_allclose(median['beta'], 0.8, rtol=rtol)
```

Comment:
the median of Beta distribution is a bit complicated to compute so it is better to skip it and use empirical test from `sample_posterior` instead.